ui/job-view/Treeherder.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { react2angular } from 'react2angular/index.es2015';
+
+import treeherder from '../js/treeherder';
+import DetailsPanel from './details/DetailsPanel';
+import PushList from './PushList';
+
+const Treeherder = (props) => {
```

Comment:
Great ideas.  I think I prefer ``JobView.jsx`` for now, since this isn't the ``App`` part of it yet, just another component.  But when we do the final changeover to it being a React app, then I'll encapsulate it with an ``index.jsx`` as you said.